src/examples/can/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"device/sam"
+	"fmt"
+	"machine"
+	"time"
+)
+
+func main() {
+	can1 := machine.CAN{Bus: sam.CAN1}
+	can1.Configure(machine.CANConfig{
+		TransferRate:   machine.CANTransferRate500kbps,
+		TransferRateFD: machine.CANTransferRate1000kbps,
+		Rx:             machine.CAN1_RX,
+		Tx:             machine.CAN1_TX,
+	})
+
+	can0 := machine.CAN{Bus: sam.CAN0}
+	can0.Configure(machine.CANConfig{
+		TransferRate:   machine.CANTransferRate500kbps,
+		TransferRateFD: machine.CANTransferRate1000kbps,
+		Rx:             machine.CAN0_RX,
+		Tx:             machine.CAN0_TX,
+	})
+
+	rxMsg := machine.CANRxBufferElement{}
+
+	for {
+		can1.Tx(0x123, 8, []byte{0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF}, false, false)
+		can1.Tx(0x789, 16, []byte{0x02, 0x24, 0x46, 0x67, 0x89, 0xAB, 0xCD, 0xEF, 0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF}, true, false)
```

Comment:
The length is specified twice: once as a parameter and once in the byte slice. It should be enough to specify it once:

```suggestion
		can1.Tx(0x123, []byte{0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF}, false, false)
		can1.Tx(0x789, []byte{0x02, 0x24, 0x46, 0x67, 0x89, 0xAB, 0xCD, 0xEF, 0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF}, true, false)
```

Reply:
Indeed, the length was extra.
I fixed it.